torchtune/_recipe_registry.py

```diff
@@ -252,6 +252,17 @@ class Recipe:
         ],
         supports_distributed=False,
     ),
+    Recipe(
+        name="dev/generate_v2",
```

Comment:
Do we wanna call it "dev/generate_v2"? Can do what we did for FSDP2 and just call "generate_v2", which is snappier from the CLI

Reply:
@pbontrager yelled at me for that

Reply:
I didn't want to call the recipe generate_v2 once it's moved out of dev. But I'm fine with not including dev in the cli name.

Reply:
could move the old recipe into dev as /generate_old

Reply:
Yeah I would just remove dev from the CLI name